Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -77,7 +77,22 @@ public static IEnumerable<FluentLayout> FullHeightOf(this UIView view, UIView pa
 			yield return view.Bottom().EqualTo().BottomOf(parent).Minus(marginValue);
         }
 
-        public static IEnumerable<FluentLayout> VerticalStackPanelConstraints(this UIView parentView, Margins margins, params UIView[] views) =>
+		public static IEnumerable<FluentLayout> FullSizeOf(this UIView view, UIView parent, nfloat? margin = null)
```

Comment:
Minor style note: for consistency, let's try to stick to using expression-bodied functions for one-liners like this